src/gettingstarted/tools.md

```diff
@@ -0,0 +1,49 @@
+# Technical Requirements: Git and SSH
+
+## Git
+
+Git is the open source version control system that is utilized by [Platform.sh](https://platform.sh/).
+
+Any change you make to your Platform.sh project will need to be committed via Git. You can see all the Git commit messages of an environment in the activity feed of the web interface.
+
+Before getting started, make sure you have it installed on your computer to be able to interact with Platform.sh.
+
+> **see also**
+>
+> * [Install Git](https://help.github.com/articles/set-up-git/)
+> * [Learn more about Git](https://git-scm.com/)
+
+
+## SSH
+
+One of the ways [Platform.sh](https://platform.sh/) keeps things secure is by using SSH behind the scenes. Users connect to their Platform.sh Git repository and to their Command Line Interface (CLI) using SSH.
```

Comment:
The CLI (`platform login`) mainly authenticates with OAuth2: it only passes through SSH when it's connecting to your Git or app servers. Suggest:

> Users connect to their Platform.sh Git repository and to their applications and services using SSH.